tools/WebStack.settings.targets

```diff
@@ -36,8 +36,8 @@
     <StyleCopTreatErrorsAsWarnings Condition=" '$(StyleCopTreatErrorsAsWarnings)' == '' ">false</StyleCopTreatErrorsAsWarnings>
     <StyleCopEnabled Condition=" '$(StyleCopEnabled)' == '' ">false</StyleCopEnabled>
 
-    <!-- Disable C# 6.0 features in Visual Studio. -->
-    <LangVersion Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">5</LangVersion>
+    <!-- Use latest C# compiler supported in Visual Studio 2019. -->
+    <LangVersion Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">9.0</LangVersion>
```

Comment:
This is a pretty big jump, is anyone relying on anything related to our LangVersion?

Reply:
We used to rely on the low version for continued compatibility w/ an outdated Visual Studio version. Now that we require a minimum VS 2019 version, this is just fine. It's also needed to avoid major additional changes in `TranscodingStream` and so on.